src/App.js

```diff
@@ -33,6 +34,9 @@ function App() {
                 <Switch>
                     <Route path="/" exact component={MainPage} />
                     <Route path="/game/:id" component={GamePage} />
+                    <Route path="/dice">
+                        <Dice />
+                    </Route>
```

Comment:
why not use it the same way as it is above: `<Route path=... component={Dice} />`?